src/System.Security.Cryptography.Encoding/tests/Oid.cs

```diff
@@ -15,6 +15,12 @@ public static void EmptyOid()
             Oid oid = new Oid("");
             Assert.Equal("", oid.Value);
             Assert.Null(oid.FriendlyName);
+
+#if netstandard17
```

Comment:
In other places we have pulled the test out into a separate file and include that just for latest version of the library. That would allow us to not have an ifdef in the test.